src/migration/1695735320395-UserPersonalizedDigest.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class UserPersonalizedDigest1695735320395 implements MigrationInterface {
+    name = 'UserPersonalizedDigest1695735320395'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
We might want to consider a index here, future proofing if we wants users on a day perhaps?

Reply:
I also think you'll need `await queryRunner.query(`ALTER TABLE "user_personalized_digest" REPLICA IDENTITY FULL`);` on this one, as it will be used in CDC correct?

Reply:
Yeah, users per day we already need, just did not add an index. Will do. 👌

Reply:
REPLICA is already set in the line below.

Reply:
Added in 3a6c0f0594e73d55fc42d4a96b7d25f518ccb91f